docs/HOWTO-add-model.md

```diff
@@ -17,7 +17,7 @@ Also, it is important to check that the examples and main ggml backends (CUDA, M
 ### 1. Convert the model to GGUF
 
 This step is done in python with a `convert` script using the [gguf](https://pypi.org/project/gguf/) library.
-Depending on the model architecture, you can use either [convert.py](../convert.py) or [convert-hf-to-gguf.py](../convert-hf-to-gguf.py).
+Depending on the model architecture, you can use either [convert-hf-to-gguf.py](../convert-hf-to-gguf.py) or [examples/convert-no-torch.py](../examples/convert-no-torch.py) (for `llama/llama2` models in `.pth` format).
```

Comment:
Given that .pth *stands for* pytorch, I think this name could be misleading. Why not call it convert-llama.py?

Reply:
`convert-legacy-llama.py` maybe? I'd like to avoid giving impression that it can be used for converting `llama 3`